packages/lesswrong/components/messaging/MessageUser.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { Components, registerComponent } from "../../lib/vulcan-lib";
+import { useLocation } from "../../lib/routeUtil";
+import { useCurrentUser } from "../common/withUser";
+import { useSingle } from "@/lib/crud/withSingle";
+import { useInitiateConversation } from "../hooks/useInitiateConversation";
+
+const MessageUserInner = ({ user }: { user: UsersMinimumInfo }) => {
+  const { Loading, PermanentRedirect, SingleColumnSection } = Components;
+
+  const { conversation, conversationLoading } = useInitiateConversation({
+    userIds: [user._id],
+  });
+
+  if (!conversation) {
+    return conversationLoading ? <Loading /> : <SingleColumnSection>Failed to create conversation: Unknown error.</SingleColumnSection>;
+  }
+
+  const url = `/inbox/${conversation._id}?from=magic_link`;
+  return <PermanentRedirect url={url} status={302} />;
+};
+
+const MessageUser = () => {
+  const currentUser = useCurrentUser();
+  const { params } = useLocation();
+  const { Loading, SingleColumnSection } = Components;
+
+  const { document: user, loading } = useSingle({
+    slug: params.slug,
+    collectionName: "Users",
+    fragmentName: "UsersMinimumInfo",
+    skip: !currentUser || !params.slug,
+  });
+
+  if (!currentUser) {
+    return <div>Log in to access private messages.</div>;
```

Comment:
Could we put this in a `<SingleColumnSection>` as well? Also I think we should give these at least a font family and font size - they are really hard to read

<img width="327" alt="Screen Shot 2024-07-29 at 5 34 01 PM" src="https://github.com/user-attachments/assets/0285cd2d-038d-4e9e-a99e-9d2d5b466eda">